### ISSUE TITLE
pdf rendering not working when branch protections are set

### ISSUE THREAD

**Issue by avahoffman** [jhudsl/intro_to_r#284]
Removed branch protections for now. But would like to figure out why it was failing even when @jhudsl-robot had permissions

For example, for Intro pdf: https://github.com/jhudsl/intro_to_r/actions/runs/3876317059/jobs/6610114522

```

Run git config --local user.email "itcrtrainingnetwork@gmail.com"
[main 813cd865] Render site
 1 file changed, 0 insertions(+), 0 deletions(-)
From https://github.com/jhudsl/intro_to_r
 * branch              main       -> FETCH_HEAD
   dfd03ff8..1ae5e9a6  main       -> origin/main
Merge made by the 'recursive' strategy.
 modules/Basic_R/lab/Basic_R_Lab_Key.html           |  20 +++---
 modules/Data_Classes/Data_Classes.html             |   2 +-
 modules/Data_Cleaning/Data_Cleaning.html           |  16 ++---
 modules/Data_Summarization/Data_Summarization.html |   2 +-
 modules/Data_Visualization/Data_Visualization.html |   2 +-
 modules/Statistics/Statistics.html                 |  12 ++--
 modules/Subsetting_Data_in_R/index_withbase.html   |   2 +-
 modules/cheatsheets/Day-1.pdf                      | Bin 115716 -> 115716 bytes
 modules/cheatsheets/Day-2.pdf                      | Bin 113865 -> 113865 bytes
 modules/cheatsheets/Day-3.pdf                      | Bin 116874 -> 116874 bytes
 modules/cheatsheets/Day-4.pdf                      | Bin 116786 -> 116786 bytes
 modules/cheatsheets/Day-5.pdf                      | Bin 109482 -> 109482 bytes
 modules/cheatsheets/Day-6.pdf                      | Bin 131071 -> 131071 bytes
 modules/cheatsheets/Day-7.pdf                      | Bin 111853 -> 111853 bytes
 modules/cheatsheets/Day-8.pdf                      | Bin 103102 -> 103102 bytes
 modules/cheatsheets/Day-9.pdf                      | Bin 102812 -> 102812 bytes
 quotes_vs_backticks.html                           |  80 ++++++++++-----------
 resources/debugging_guide.pdf                      | Bin 170097 -> 170097 bytes
 resources/good-scientific-coding-practices.pdf     | Bin 155392 -> 155392 bytes
 resources/rtools_windows.pdf                       | Bin 74301 -> 74301 bytes
 20 files changed, 68 insertions(+), 68 deletions(-)
remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To https://github.com/jhudsl/intro_to_r
 ! [remote rejected]   main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/jhudsl/intro_to_r'
No changes to push
```

**Comment by avahoffman** [comment]
I think this will be addressed by #289 but want to check that main renders correctly.

**Comment by avahoffman** [comment]
Credentials were not persisting. Fixed now!

https://github.com/jhudsl/intro_to_r/blob/b9acf13e1fdf6cd6e758feb7cbb599d395e67505/.github/workflows/render-site.yml#L166
